js/forum/src/main.js

```diff
@@ -27,6 +29,12 @@ app.initializers.add('absagar-auth-testbook', function() {
       items.remove('account');
     }
   });
+
+
+  extend(AvatarEditor.prototype, 'view', function(obj) {
+      obj.children[1].attrs.style="visibility: hidden";
```

Comment:
this becomes very error prone this way and is tightly coupled with the dom structure of AvatarEditor. We can simple override the view to return this : 
      ```
<div className={'AvatarEditor Dropdown ' + this.props.className + (this.loading ? ' loading' : '')}>
        {avatar(user)}
      </div>
```